### DISCUSSION TITLE
GitHub API Status Badge in `README.md` 's

### DISCUSSION THREAD

**Discussion by oscard0m** [octokit/octokit.js#2097] [Ideas]
From [this tweet](https://twitter.com/gr2m/status/1394686065085075458?s=20) from @gr2m an idea came to my mind.

What would be your opinion on having a satus Badge of different GitHub Services from https://www.githubstatus.com/
 
I just read about this https://github.com/upptime/upptime but maybe the details should be discussed in an issue if you like the idea.

**Comment by gr2m** [comment]
Very cool idea! Sounds like it might be a good addition to http://shields.io?
